test/project.e2e-spec.ts

```diff
@@ -535,6 +536,9 @@ describe('Project e2e', () => {
           input: {
             sort: 'sensitivity',
             order,
+            filter: {
+              mine: true,
+            },
```

Comment:
Excluding this shouldn't fail the test

Reply:
Not sure why it fails without mine filter.

Reply:
if we don't add mine filter, it returns all the projects like below.
![image](https://user-images.githubusercontent.com/4212008/97469198-dc2fbb00-1991-11eb-8cd7-2f9b0ca08c73.png)